waiter/integration/waiter/kubernetes_scheduler_integration_test.clj

```diff
@@ -258,6 +258,22 @@
           (let [service-account (get-pod-service-account waiter-url current-user)]
             (is (= current-user service-account))))))))
 
+(deftest ^:parallel ^:integration-fast test-kubernetes-grace-period-disabled
```

Comment:
Why is this a Kubernetes test? Shouldn't this work on all platforms?

Reply:
In addition to this check, should we also have a check that uses the `--enable-status-change` that you added a while back? We could switch the app to unhealthy and assert that the connection times out (with an override for the timeout so it doesn't actually wait a whole 5 minutes) rather than getting routed to a new instance.

Reply:
Moved the test and confirmed that requests get queue timeout due to lack of a healthy instance.